src/main/scala/org/broadinstitute/dsde/workbench/sam/config/KmsConfig.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsde.workbench.sam.config
+
+/**
+  * created by mtalbott 1/25/18
+  *
+  * @param project google project for the key
+  * @param location location for the key
+  * @param keyRingId name of the key ring that contains the key
+  * @param keyId name of the key
+  */
+
+final case class GoogleKmsConfig(
+                                  project: String,
+                                  location: String,
+                                  keyRingId: String,
+                                  keyId: String
```

Comment:
seems like there're custom types for `project`, `location`, `keyRingId` etc, why not use those? that way, you won't need to rewrap [here](https://github.com/broadinstitute/sam/pull/275/files#diff-ab230e3b899f0b14c89b226c3c761c0eR138). It's also more type safe